client/src/components/basic/rank/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { combineClasses } from 'helpers/combineClasses.helper';
+import React, { FC } from 'react';
+import styles from './rank.module.scss';
+
+function mapRankToClass(rank: number): string | undefined {
+	if ([1, 2, 3].includes(rank)) {
+		return styles.hard;
+	} else if ([4, 5, 6].includes(rank)) {
+		return styles.medium;
+	} else if ([7, 8, 9].includes(rank)) {
+		return styles.easy;
+	}
+}
```

Comment:
1. Such function better to move to helpers file (in the same folder). so basic/rank/helpers.ts
2. This function will not work properly when it's using for honor value (not rank)
3. There should be default theme. 
4. `else if` is not needed for this function because you have `return` in each if condition
5. [1, 2, 3], [4, 5, 6] etc.. Better to move to config file. It should be some general application config, but we can start from  basic/rank/config.ts

Reply:
No need to change, just have in mind :)
You can use `find` method for this function:
```
return [ [1, 2, 3], ..... ].find((themeValues) => themeValues.includes(rank) ) || "default value"
```

it's better because it's easier to add more preferences to config instead of adding more and more `if` conditions